apps/rag/app/rag.py

```diff
@@ -5,19 +5,24 @@
 
 
 class QAChain:
-    def __init__(self, retriever, context_builder, llm):
+    def __init__(self, retriever, context_builder, llm, rewrite_query=False):
         self.retriever = retriever
         self.context_builder = context_builder
         self.llm = llm
         self.separator = context_builder.separator
         self.separator_length = len(self.separator)
+        self.rewrite_query = rewrite_query
 
     async def run(self, query: str, history: str) -> Tuple[str, str]:
-        rewritten_query = await self._rewrite_query(query, history)
-        logger.debug(f"Rewritten query: {rewritten_query}")
-        if not rewritten_query:
-            logger.error(
-                "Rewritten query is empty, using original query; check that the LLM is working.")
+        if self.rewrite_query:
+            rewritten_query = await self._rewrite_query(query, history)
+            logger.debug(f"Rewritten query: {rewritten_query}")
+            if not rewritten_query:
+                logger.error(
+                    "Rewritten query is empty, using original query; check that the LLM is working.")
+                rewritten_query = query
+        else:
+            logger.info("Not rewriting query")
```

Comment:
```suggestion
            logger.debug("Not rewriting query")
```
Not doing it shouldn't be noisier than doing it.